### ISSUE TITLE
How do I do a backslash in a code block?

### ISSUE THREAD

**Issue by ctaggart** [tpetricek/FSharp.Formatting#327]
I'm editing the documentation for the main code block on the main page of
http://ctaggart.github.io/SourceLink/

I keep getting this:
![image](https://cloud.githubusercontent.com/assets/80104/8495216/9c2c8548-2120-11e5-8637-5c8076e29471.png)

When what I want is this:
```
SourceLink.exe index `
-pr .\src\fsharp\FSharp.Core\FSharp.Core.fsproj `
-pp Configuration Release `
-u 'https://raw.githubusercontent.com/Microsoft/visualfsharp/{0}/%var2%'
```

**Comment by tpetricek** [comment]
Try using:

    Some _markdown_ stuff

        [lang=text]
        some command

    some more markdown

**Comment by tpetricek** [comment]
But I'm not sure if that helps - if no, feel free to report back. We might have broken something :)

**Comment by matthid** [comment]
I'm pretty sure code blocks now work as specified in common-mark, since https://www.nuget.org/packages/FSharp.Formatting/2.13.0. Feel free to comment if your problem is not solved with 2.13
